validator/main.go

```diff
@@ -304,6 +304,16 @@ contract in order to activate the validator client`,
 		}
 		flags.ComplainOnDeprecatedFlags(ctx)
 
+		if !ctx.Bool(featureconfig.AltonaTestnet.Name) &&
```

Comment:
Same comment here. This is duplicated code that could be refactored and reused in all three processes.